### PR TITLE
PBM-435 pbm status command

### PR DESCRIPTION
**Pull request by dAdAbird**
https://jira.percona.com/browse/PBM-435

### CODE REVIEW

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| mongodb 3.6 | failed |
| mongodb 4.0 | failed |
| mongodb 4.2 | failed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| mongodb 3.6 | failed |
| mongodb 4.0 | failed |
| mongodb 4.2 | failed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| mongodb 3.6 | failed |
| mongodb 4.0 | failed |
| mongodb 4.2 | failed |

**Review by akira-kurogane — APPROVED** [review]
Otsukare-san! LGTM

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| mongodb 3.6 | failed |
| mongodb 4.0 | failed |
| mongodb 4.2 | failed |

File: cmd/pbm/status.go [hunk 1]
```diff
@@ -0,0 +1,560 @@
...
+		// Just check if all locks are for the same op
+		//
+		// It could happen that the healthy `lk` became stale by the time this check
+		// or the op was finished and the new one was started. So the `l.Type != lk.Type`
+		// would be true but for the legit reason (no error).
+		// But chances for that are quite low and on the next run of `pbm status` everythin
+		//  would be ok. So no reason to complicate code to avoid that.
+		if lk != nil && l.OPID != lk.OPID {
+			if err != nil {
+				return nil, errors.Errorf("conflicting ops running: [%s/%s::%s-%s] [%s/%s::%s-%s]",
```
[the remark this answers is not in the document]
Comment (dAdAbird) [reply cmd/pbm/status.go]: done!

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| mongodb 3.6 | failed |
| mongodb 4.0 | failed |
| mongodb 4.2 | failed |

**Comment by JNKPercona** [comment]
| Test name  | Status |
| ------------- | ------------- |
| mongodb 3.6 | failed |
| mongodb 4.0 | failed |
| mongodb 4.2 | failed |

**Pull request merged**